pkg/util/k8scmcache.go

```diff
@@ -128,7 +128,7 @@ func (k8scm *K8sCMCache) Create(identifier string, data interface{}) error {
 	}
 	dataJSON, err := json.Marshal(data)
 	if err != nil {
-		return errors.Wrap(err, "k8s-cm-cache: marshal error")
+		return errors.Wrapf(err, "k8s-cm-cache: JSON marshaling failed for configmap %s: %v", identifier, err)
```

Comment:
`Wrapf` already wrap  `err`  with string i think we can skip  adding err at last

simply it will become 
`errors.Wrapf(err, "k8s-cm-cache: JSON marshaling failed for configmap %s:",identifier)`